build/travis/before_install.sh

```diff
@@ -10,4 +10,6 @@ if [ "$TEST_DIR" = "server" ]; then
 fi
 
 gem update --system
-gem install bundler-audit --no-ri --no-rdoc
\ No newline at end of file
+gem install bundler-audit --no-ri --no-rdoc
+stty cols 157
```

Comment:
#3331 will fake the `TTY::Screen.width` for all specs: https://github.com/kontena/kontena/pull/3331/files#diff-cf82573a2ae83c839d397f095fdc8b33

Reply:
Yep that's better than messing with the travis shell script.

Reply:
@kke and @SpComb, agree - stubbing `TTY::Screen` is way more portable than `before_install.sh`. Therefore I'm reverting my changes and back-porting yours to unblock my CI runs.

Reply:
✅ RESOLVED